src/encoded/static/components/home.js

```diff
@@ -15,11 +16,11 @@ dayjs.extend(utc);
 // Convert the selected organisms and assays into an encoded query.
 function generateQuery(selectedOrganisms, selectedAssayCategory) {
     // Make the base query.
-    let query = selectedAssayCategory === 'COMPPRED' ? '?type=Annotation&encyclopedia_version=ENCODE+v4' : '?type=Experiment&status=released';
+    let query = selectedAssayCategory === 'COMPPRED' ? '?type=Annotation&encyclopedia_version=ENCODE+v5' : '?type=Experiment&status=released';
```

Comment:
If we plan to update ENCODE version on every update, would it be better if 'v5' is a constant declared at the top of the page?